packages/node_modules/pouchdb-fetch/src/fetch.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+import nodeFetch from 'node-fetch';
+import fetchCookie from 'fetch-cookie';
```

Comment:
Shouldn't that be `'fetch-cookie/node-fetch'` to support redirects ?
cf. [fetch-cookie: cookies and redirection](https://github.com/valeriangalliat/fetch-cookie#cookies-and-redirection)

Reply:
I think we can take a look at that in a follow up, would be curious as to why that isnt the default

Reply:
@daleharvey Well, they explain, on the link that I referenced, that they have to rely on APIs that only `node-fetch` provides. More precisely:
> the generic fetch API does not allow any way to hook into redirects

Reply:
Ah ok, since this is green gonna marge as is and if you wanna file and PR that as a follow up it would be awesome, thanks